psutil/_pslinux.py

```diff
@@ -159,8 +159,14 @@ class IOPriority(enum.IntEnum):
                 'read_time', 'write_time',
                 'read_merged_count', 'write_merged_count',
                 'busy_time'])
-popenfile = namedtuple(
-    'popenfile', ['path', 'fd', 'position', 'mode', 'flags'])
+
+popenfile = namedtuple('popenfile', ['path', 'fd', 'position', 'mode', 
+    'flags', 'mnt_id', 'anon_fields'])
+
```

Comment:
Added the `mnt_id` field and a `anon_fields` which defaults to an empty `dict` if a regular file.